src/components/ListItem.jsx

```diff
@@ -4,35 +4,34 @@ import { updateItem } from '../api/firebase';
 
 export function ListItem({ item, listId }) {
 	const [checked, setChecked] = useState(false);
-	const { name, id, dateLastPurchased } = item;
 
 	const checkItem = (e) => {
-		updateItem(listId, id);
+		updateItem(listId, item);
 	};
-
 	useEffect(() => {
-		if (dateLastPurchased) {
-			const lastPurchasedDate = dateLastPurchased.toDate();
-			const hoursPlus24 = lastPurchasedDate.getHours() + 24;
+		if (item.dateLastPurchased) {
+			const lastPurchasedDate = item.dateLastPurchased.toDate();
+			const twentyFourHrs = 24;
```

Comment:
We had addressed the removal of magic numbers last issue but it was accidentally refactored.

Reply:
Good looking out for this.